raiden/channel/netting_channel.py

```diff
@@ -481,7 +155,6 @@ def __init__(
             block_number):
 
         if settle_timeout <= reveal_timeout:
-            # reveal_timeout must be a fraction of the settle_timeout
             raise ValueError('reveal_timeout can not be larger-or-equal to settle_timeout')
```

Comment:
I think it is weird to do the test in the other direction than the error message, but it is correct still.